templates/email/messages/berth_invoice_en.html

```diff
@@ -17,7 +17,7 @@ <h3>Price of the boat berth</h3>
 <table style="width: 100%">
     <tr>
         <td colspan="2">
-            Basic price ({{ order.product.price_group.name }}):
+            Basic price ({{ order.product.display_dimensions }}):
```

Comment:
The text here seems a bit weird, but I guess it's intentional.

Reply:
You mean the `Basic price` itself?

Reply:
Right, I guess we could go with `Base price`, would be a bit clearer.

Reply:
The `order.product.display_dimensions` gives the range in meters and the customer isn't really aware of the pricing scheme being used. That's why I think it might be a bit confusing.

Reply:
Gotcha. We are going to inform the customers about the pricing changes, so in my mind it would make sense to display the price so they can corroborate that it's the right one.

(The full answer:)
> Yes, the idea is that customers (in the mailing list) know about the new pricing and about the need to check the invoice before they pay it before the actual invoices are sent. And luckily that's what people usually do, check the amount before they pay